lxd/main_init.go

```diff
@@ -249,9 +253,10 @@ func cmdInit() error {
 			}
 		}
 
-		if storageSetup && storageBackend == "zfs" {
+		if storageSetup && storageBackend != "dir" {
```

Comment:
Hm, why? I just created two dir pools with two lxd init runs named `bla` and `blu`. This condition only excludes block devices and loop files for dir.